### ISSUE TITLE
GUN + NKN

### ISSUE THREAD

**Issue by omani** [amark/gun#1310]
hey folks,
coming from the NKN community, I always wanted to have some sort of a distributed p2p db for the web or web3.0 applications. I spent some time to dive into GUN and realized that AXE was invented to have an incentive for relayers to relay data.

I heard @amark in the embedded youtube video at https://gun.eco/docs/AXE that he considered other blockchains (especially Bitcoin) before AXE was invented.

so here is my question:
have you considered NKN? (nkn.org) (github.com/nknorg/nkn)? it has payment channels (nanopay), a free transit p2p network and miners mine (mining happens by relaying network data, proof of relay) the mainnet coin NKN. currently there are more than 60k mining nodes running the NKN network. IMO the whole NKN ecosystem is missing the most important part, the p2p database to build apps.

so basically everything he explained about AXE in that video reminded me of NKN, because NKN has all of that already.

I wonder what you guys (and the GUN community) think about this.

**Comment by amark** [comment]
If NKN people want to come help or build on GUN, sure, but I just don't have bandwidth elsewise, sorry.
